tests/unittest/pywbem/test_mof_compiler.py

```diff
@@ -80,7 +78,7 @@ def moflog(msg):
         self.mofcomp = MOFCompiler(
             MOFWBEMConnection(),
             search_paths=[TEST_DMTF_CIMSCHEMA_MOF_DIR],
-            verbose=False,
+            verbose=True,
```

Comment:
This should be reset to False as it displays a lot of stuff now.

Reply:
Done